robocopkaka/hospital
# Ft admins can act on appointments 161097353

#### What does this PR do?
Allows admins approve/decline appointments
#### Description of Task to be completed?
* Give admins the ability to approve appointments
* Give admins the ability to decline appointments
* Send emails to patients when their appointments are either approved or declined
#### How should this be manually tested?
* Log in as an admin
* Click on the `Appointments` tab and select `View pending appointments`
* To approve an appointment, click on the green check icon and select the doctor to handle the appointment
* To decline an appointment, click on the red cancel icon and then click yes on the confirmation dialogue
* Approved/declined appointments should not show up on the list of pending appointments
#### What are the relevant pivotal tracker stories?
[#161097353](https://www.pivotaltracker.com/story/show/161097353)
#### Any background context you want to add?
N/A
#### Screenshots
![screen shot 2018-11-06 at 2 16 54 pm](https://user-images.githubusercontent.com/11176000/48066722-a60a3700-e1ce-11e8-94df-68a11d66a5b3.png)
